### ISSUE TITLE
Proper CMake support

### ISSUE THREAD

**Issue by Quincunx271** [eranpeer/FakeIt#139]
It would be nice to be able to consume this library from cmake with syntax like:

```cmake
find_package(FakeIt REQUIRED)

# ...

target_link_libraries(MyTarget FakeIt::FakeIt)
```

Some options I know of to make this possible:

* Write a *FindFakeIt.cmake* file.
* Flesh out *CMakeLists.txt* to be able to build and install FakeIt

**Comment by libenike** [comment]
I have implemented cmake support of FakeIt.
In my fork you can link FakeIt to a project by this way:

```
find_package(FakeIt REQUIRED)

# ...

target_link_libraries(MyTarget FakeIt::FakeIt)
```

Link to the [pull request](https://github.com/eranpeer/FakeIt/pull/192).
Link to a simple [project](https://github.com/libenike/FakeItUsageExample.git) which use FakeIt. Here you can find detailed building instructions for the project.

**Comment by FranckRJ** [comment]
Fixed by #192 and #311, it's already in master.
